### ISSUE TITLE
[Request] per torrent speed limit

### ISSUE THREAD

**Issue by noraj1337** [jfurrow/flood#212]
I would be great if we can be able to set a speed limit **per torrent** additionnaly to **global** speed limit.

**Comment by jfurrow** [comment]
I agree and I need to research how this is currently handled in ruTorrent.

**Comment by DavePS** [comment]
@jfurrow this is what I found about rTorrent's XMLRPC commands : https://github.com/mdevaev/emonoda/wiki/rTorrent-XMLRPC-Reference

**Comment by noraj1337** [comment]
Maybe `set_max_downloads_div` and `set_max_uploads_div` ?
